docs/content/deprecated.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Deprecated
+---
+
+## Deprecated in v0.0.28
+
+### System Arguments
+
+These presentational system arguments were deprecated in favor of functional color names.
+
+| argument | value | replacement |
+| :- | :- | :- |
+| `color:` | `:blue` | `:link` |
+| `color:` | `gray_dark:` | `:primary` |
+| `color:` | `gray:` | `:secondary` |
+| `color:` | `gray_light:` | `:tertiary` |
+| `color:` | `green:` | `:success` |
+| `color:` | `yellow:` | `:warning` |
+| `color:` | `red:` | `:danger` |
```

Comment:
I wanted to create a url we could link to with more instructions https://view-components-git-linterfordeprecatingarguments-primer.vercel.app/view-components/deprecated

I tried to leave this in a format like this so we could repeat for other things like components, arguments.

```
Version
- Types of deprecations
   - Table of deprecations
Version
- Types of deprecations
   - Table of deprecations
Version
- Types of deprecations
   - Table of deprecations
```

Reply:
@camertron @manuelpuyol what do you think about adding the CHANGELOG to the docs site and just publishing this as part of the release notes? We do something similar for ViewComponent.

Reply:
Hmm not sure I understand, can you expand a little more on what that would mean? 🤔